src/inputs/button/button.jsx

```diff
@@ -300,6 +305,10 @@ class Button extends React.PureComponent {
         }
     }
 
+    onMouseDown(event) {
+        event.preventDefault();
+    }
```

Comment:
Same question here.

Reply:
I found an answer in a GitHub thread somewhere that doing this will block the client from bubbling up the event to focus the input. I don't believe this is the best answer to the problem, but I believe design needs to design the different states. For now, it works in Chrome, Safari, and Firefox: when tabbing the input is outlined and when clicking input it is not outlined.